src/site/content/en/blog/intersectionobserver-v2/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Trust is Good, Observation is Better—Intersection Observer v2
+subhead: |
+  Intersection Observer v2 adds the capability to not only observe intersections per se, but to also
+  detect if the intersecting element was visible at the time of intersection.
+authors:
+  - thomassteiner
+description: |
+  Intersection Observer v2 adds the capability to not only observe intersections per se, but to also
+  detect if the intersecting element was visible at the time of intersection.
+date: 2019-02-15
+updated: 2021-02-26
+tags:
+  - blog
+  - intersectionobserver
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/wUCO2CgK5UOz6URpXEtd.jpeg
+alt: Eye of a cat looking through a gap in a fence.
+---
+
+Intersection Observer&nbsp;v1 is one of those APIs that's probably universally loved, and, now that
+[Safari supports it](https://webkit.org/blog/8582/intersectionobserver-in-webkit/) as well,
+it's also finally universally usable in all major browsers. For a quick refresher of the API,
+I recommend watching [Surma](../../../resources/contributors/surma)'s
+[Supercharged Microtip](https://www.youtube.com/embed/kW_atFXMG98) on Intersection
+Observer&nbsp;v1—also embedded below for your viewing pleasure—or reading Surma's in-depth
```

Comment:
Since this is an old article, I'm skipping a lot of writing feedback I would normally give. This part of the sentence loses me completely.

Reply:
Massively simplified. These horrible [intertwingled](https://www.urbandictionary.com/define.php?term=intertwingled) sentences…